Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TextBoxMask/TextBoxMaskPage.xaml

```diff
@@ -0,0 +1,51 @@
+﻿<Page
```

Comment:
Fixed, I downloaded xaml styler and applied the default rules on the page.
We should add Xaml Styler to the contribution section